salesagility/SuiteCRM
# Composerify Smarty

## Description
Fixes #6719.

This keeps the `Smarty/plugins/` directory for now since I didn't want to move it and cause more breakage.

This removes the security-advisories package because it was preventing me from installing Smarty (unless I upgraded to 3.1, which would almost definitely cause significant breakage). I realize that's not ideal, but we're not changing anything with, if anything we're _improving_ the security of the CRM by upgrading from 2.6.29 to 2.6.31, and moving it to Composer so it's easier to upgrade.

## How To Test This
- Make sure the tests pass.
- Make sure that you can run `composer install`.
- Make sure the CRM's main pages still work and render correctly. 

## Types of changes

This is _technically_ a breaking change if anyone was trying to require a file from `include/Smarty/` directly, but other than that this shouldn't be a problem.


### Final checklist
